src/APICalls/APICalls.js

```diff
@@ -1,5 +1,14 @@
+const endpoints = {
+  playersProd: 'https://fore-finder-be.herokuapp.com/api/v1/players',
+  playersDev: 'http://8f05812ea9bf.ngrok.io/api/v1/players',
+  coursesProd: 'https://fore-finder-be.herokuapp.com/api/v1/courses',
+  coursesDev: 'http://8f05812ea9bf.ngrok.io/api/v1/courses',
+  eventsProd: 'https://fore-finder-be.herokuapp.com/api/v1/events',
+  eventsDev: 'http://8f05812ea9bf.ngrok.io/api/v1/events'
+}
```

Comment:
This is a great idea to change this as the endpoints are constantly changing with ngrok

Reply:
@jon-schlandt deserves the credit for that one